packages/core/src/models/lessonPlans.ts

```diff
@@ -232,27 +241,37 @@ export class LessonPlans {
       },
     });
 
-    for (const key of Object.keys(json).filter(
-      (k) => !["subject", "keyStage"].includes(k),
-    )) {
-      // Ignore empty values
-      if (["", "None", undefined, null].includes(json[key])) {
-        break;
```

Comment:
Was breaking rather than continuing a bug?

Reply:
Yes - it should be continue here or we are ignoring lesson plan parts